app/main/posts/posts-routes.js

```diff
@@ -1,155 +1,158 @@
 module.exports = [
     '$stateProvider',
+    '$urlMatcherFactoryProvider',
 function (
-    $stateProvider
+    $stateProvider,
+    $urlMatcherFactoryProvider
 ) {
+    $urlMatcherFactoryProvider.strictMode(false);
```

Comment:
Didn't we already set this?

Reply:
yea but you need to do it for each routing file or it doesn't care for it. That's why we have it in post-routes and common routes , etc.